pydy/viz/static/js/dyviz/param_editor.js

```diff
@@ -126,7 +124,7 @@ DynamicsVisualizer.ParamEditor = Object.extend(DynamicsVisualizer, {
 
         jQuery.extend(true,self.model.objects[int_id],updated_object);
         self.Scene.addObjects();
-        self.Scene.addCameras();
+        //self.Scene.addCameras();
```

Comment:
Yes! It will help in future if we need to debug. Tarun's commented code helped me a lot finding my way.
Yes custom cameras now work, this part is run when we click "Apply" in editobjects. That's why we don't need to set camera here.